src/playlistServices/playtistservice.js

```diff
@@ -0,0 +1,65 @@
+import axios from "axios"
+
+const createPlaylist = async (title) => {
+    try{
+        const {data:{playlists}} = await axios.post("/api/user/playlists", {playlist: {title: title}}, 
+        {
+            headers: { authorization: localStorage.getItem("token") },
+        })
+        localStorage.setItem("playlists", JSON.stringify(playlists));
+        return playlists;
+    }catch(error){
+        console.log("Playlist creation error -", error);
```

Comment:
remove console log and show error to user via some toast

Reply:
yes i had kept this to add toast as a feature